pyproject.toml

```diff
@@ -56,7 +56,7 @@ dependencies = [
     "pydantic",
     "scipy",
     "torch",
-    "transformers",
+    "transformers<=4.52.4",
```

Comment:
Better to restrict to minor version: `"transformers~=4.52"`.

Reply:
Nope, such restriction leads to downloading transformers version 4.53.2, which is broken